expressjs-backend/routes/userRoutes.js

```diff
@@ -460,6 +460,29 @@ router.post('/change-profile-image', verifyToken, upload.single('file'), async (
   }
 });
 
+router.get('/api/users/:userId', async (req, res) => {
```

Comment:
some of this is Id and some is ID, let's just make sure D is always capital

Reply:
Agreed, should be consistent

Reply:
I will go through and fix the routes. I just generic ones down for now, for testing purposes.